src/main/java/com/thealgorithms/bitmanipulation/SingleBitOperator.java

```diff
@@ -0,0 +1,32 @@
+package com.thealgorithms.bitmanipulation;
+
+/*
+ * Author: lukasb1b (https://github.com/lukasb1b)
+ */
+
+public final class SingleBitOperator {

```

Comment:
As suggested [earlier](https://github.com/TheAlgorithms/Java/pull/4415#discussion_r1338339851):
```suggestion
public final class SingleBitOperator {
    private SingleBitOperator() {
    }
```